### PR TITLE
WIP: Add versions specific for RHODS

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

**Comment by goern** [comment]
I like this, @fridex could you write up a short user story or demo why and what we do here?

**Comment by fridex** [comment]
> I like this, @fridex could you write up a short user story or demo why and what we do here?

The demo is available at - https://www.youtube.com/watch?v=4vIpVxe7-9c. The user story would be something like:

As a product release manager, I would like to make sure all the container images are shipped with a tested set of dependencies so that I know all the container images built work in a cluster without a need to manually manage lock files in all the repositories (centralize knowledge about dependencies that are claimed to work). To test the container images with another set of dependencies, I would like to just retrigger the build process of all the container images with a different set of environment variables (labels) supplied (e.g. different product version release).

File: prescriptions/_rhods/beautifulsoup4.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        version: '!=4.8.2'
+    name: RHODSBeautifulsoup4Sieve
+    run:
+      log:
+        message: Removing package 'beautifulsoup4' as version does not match product
+          version expected '==4.8.2'
+        type: WARNING
+      stack_info:
+      - link: https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-data-science
+        message: Removing package 'beautifulsoup4' as version does not match product
```
[the remark this answers is not in the document]
Comment (fridex) [reply prescriptions/_rhods/beautifulsoup4.yaml:16]: > Can we use this as a user-facing justification on a HTML page describing the resulting container image

Yes, I think so.

**Pull request closed without merging**